pkgs/by-name/vu/vuetorrent/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildNpmPackage rec {
+  pname = "vuetorrent";
+  version = "2.18.0";
+
+  src = fetchFromGitHub {
+    owner = "VueTorrent";
+    repo = "VueTorrent";
+    rev = "refs/tags/v${version}";
```

Comment:
```suggestion
    tag = "v${version}";
```

Reply:
On a recent enough nixpkgs, which includes #355973.

Reply:
Rebased PR onto latest nixpkgs & updated to use the `tag` attr.